2.cellprofiler_analysis/nf1_analysis.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# activate the main conda environment
+eval "$(conda shell.bash hook)"
+conda activate nf1_cellpainting_data
```

Comment:
What else does the `eval` line do besides activation? Should this be documented here to delineate what's happening?

Reply:
Ahh probably, I was getting errors trying to just use the conda activate line so this was the solution to allow for the environment to activate from the sh file.